tests/unit/deferred.py

```diff
@@ -196,15 +206,23 @@ def test_apply_networking(switch, network, fresh_database):
     try:
         deferred.apply_networking()
     except SwitchError:
-        with app.app_context():
-            pending_action = db.session \
-                .query(model.NetworkingAction). \
-                order_by(model.NetworkingAction.id).first()
-            current_count = db.session \
-                .query(model.NetworkingAction).count()
-
-        # assert that there's only 1 remaining action and that has type
-        # revert_port. confirming that previous modify_port operations were
-        # completed successfully.
+        local_app = Flask(__name__.split('.')[0])
+        uri = config.cfg.get('database', 'uri')
+        local_app.config.update(SQLALCHEMY_TRACK_MODIFICATIONS=False)
+        local_app.config.update(SQLALCHEMY_DATABASE_URI=uri)
+        local_db = SQLAlchemy(local_app)
```

Comment:
Should probably factor this out into its own function; this is copypasta from above.